hexa/workspaces/graphql/schema.graphql

```diff
@@ -854,4 +854,122 @@ extend type Mutation {
 extend type MePermissions {
   # Indicates whether the user can create a workspace.
   createWorkspace: Boolean!
-}
\ No newline at end of file
+}
+
+"""
+DHIS2 metadata item
+"""
+type DHIS2MetadataItem {
+    id: String!     # Unique identifier
+    name: String!   # Name of the metadata item
+}
+enum DHIS2ConnectionError {
+    CONNECTION_ERROR
```

Comment:
I see what you mean by "CONNECTION_ERROR" but maybe we could use another wording like "REQUEST_ERROR" no?